File: 이코테/곱하기혹은더하기.cpp
#include <iostream>

using namespace std;

int main(void)
{
	
	string str;
	int num;

	cin>>str; 


	int result = str[0] ^ '0';

	for(int i=1; i<str.length(); i++)
	{
		num = str[i] ^ '0';

		if( result > 0 && num > 0 )
			result *= num;

		else
			result += num;
	}

	cout<<"result : "<<result<<endl;

	return 0;	
}

File: 이코테/곱하기혹은더하기.py
N = input()

result = int(N[0])


for i in range(1, len(N)):
	
	print('흠')

	num = int(N[i])

	if( result > 0 and num > 0 ):
		result *= num

	else:
		result += num
	
	

print('result : {}'.format(result))
